.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
```

Comment:
We can put `screenshots` under `Actual behavior` section as [Antrea](https://github.com/vmware-tanzu/antrea/blob/master/.github/ISSUE_TEMPLATE/bug_report.md) does for users to add some texts or logs.

Reply:
Yes makes sense. Changed it accordingly.